src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberMethodSymbol.cs

```diff
@@ -1399,6 +1402,16 @@ private void DecodeWellKnownAttributeAppliedToReturnValue(ref DecodeWellKnownAtt
             {
                 arguments.GetOrCreateData<ReturnTypeWellKnownAttributeData>().HasNotNullAttribute = true;
             }
+            else if (attribute.IsTargetAttribute(this, AttributeDescription.NotNullIfNotNullAttribute))
+            {
+                arguments.GetOrCreateData<ReturnTypeWellKnownAttributeData>().AddNotNullIfParameterNotNull(DecodeNotNullIfNotNullAttribute(attribute));
+            }
+        }
+
+        internal static string DecodeNotNullIfNotNullAttribute(CSharpAttributeData attribute)
```

Comment:
>DecodeNotNullIfNotNullAttribute [](start = 31, length = 31)

Consider moving to a shared location, perhaps `Microsoft.CodeAnalysis.CSharp.AttributeDataExtensions`.

Reply:
Pending